src/js/api/build.js

```diff
@@ -126,6 +126,145 @@ ripe.Ripe.prototype.getBuildArtifactP = function(name, version, options) {
     });
 };
 
+/**
+ * Server side callback method to be called for situations where the initials
+ * or engraving values were changed.
+ * This method allows the change of the current context of execution based on
+ * a server side implementation of the 3DB's business logic.
+ *
+ * @param {Object} options An object with options, such as:
+ *  - 'group' - The name of the group that is going to be changed
+ *  - 'value' - The initials value to be changed
+ *  - 'engraving' - The engraving value to be changed
+ * @param {Function} callback Function with the result of the request.
+ * @returns {Promise} Resulting information for the callback execution.
+ */
+ripe.Ripe.prototype.onInitialsP = function(options, callback) {
+    return new Promise((resolve, reject) => {
+        this.onInitials(options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request));
+        });
+    });
+};
+
+/**
+ * Updates a build by brand name and version.
+ *
+ * @param {String} name The name of the brand of the build.
+ * @param {Object} options An object of options to configure the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.updateBuild = function(name, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const url = `${this.url}builds/${name}/update`;
+
+    options = Object.assign(options, {
+        url: url,
+        method: "GET",
+        auth: true
+    });
+    options = this._build(options);
+    return this._cacheURL(options.url, options, callback);
+};
+
+/**
+ * Updates a build by brand name and version.
+ *
+ * @param {String} name The name of the brand of the build.
+ * @param {Object} options An object of options to configure the request.
+ * @returns {Promise} The build update (as a promise).
+ */
+ripe.Ripe.prototype.updateBuildP = function(name, options) {
+    return new Promise((resolve, reject) => {
+        this.updateBuild(name, options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request));
+        });
+    });
+};
+
+/**
+ * Uninstalls a build by brand name and version.
+ *
+ * @param {String} name The name of the brand of the build.
+ * @param {Object} options An object of options to configure the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.uninstallBuild = function(name, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const version = options.version === undefined ? "latest" : options.version;

```

Comment:
Actually, unlike install, uninstall shouldn't default to `latest` (check https://github.com/ripe-tech/ripe-core/blob/master/src/ripe_core/controllers/api/build.py#L65).